### PR TITLE
Add support for Symfony 7

### PR DESCRIPTION
**Pull request by loic425**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -13,23 +13,30 @@ on:
...
     tests:
         runs-on: ubuntu-latest
+        continue-on-error: ${{ matrix.experimental }}
         name: "PHP ${{ matrix.php }}${{ matrix.symfony != '' && format(', Symfony {0}', matrix.symfony) || '' }}${{ matrix.currency-bundle != '' && format(', CurrencyBundle {0}', matrix.currency-bundle) || '' }}${{ matrix.postgres != '' && format(', PostgresSQL {0}', matrix.postgres) || '' }}"
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.0", "8.1"]
+                php: ["8.1", "8.2", "8.3"]
                 postgres: [""]
```
Comment (diimpp) [inline .github/workflows/build.yml:21]: Outside of scope of current PR, but why this `build.yml` uses postgresql and not mysql as DB for tests?

**Review by diimpp — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (loic425) [reply .github/workflows/build.yml:21]: Build is using postgresql, cause there was a bug with postgresql on StringFilter. But the default database manager is sql lite by default in the CI and on the test application.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -13,23 +13,30 @@ on:
 jobs:
     tests:
         runs-on: ubuntu-latest
+        continue-on-error: ${{ matrix.experimental }}
         name: "PHP ${{ matrix.php }}${{ matrix.symfony != '' && format(', Symfony {0}', matrix.symfony) || '' }}${{ matrix.currency-bundle != '' && format(', CurrencyBundle {0}', matrix.currency-bundle) || '' }}${{ matrix.postgres != '' && format(', PostgresSQL {0}', matrix.postgres) || '' }}"
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.0", "8.1"]
+                php: ["8.1", "8.2", "8.3"]
```
Comment (NoResponseMate) [inline .github/workflows/build.yml:20]: Since we're allowing installation with PHP 8.0, I think we should leave at least a single build for it.

Comment (GSadee) [reply .github/workflows/build.yml:20]: I would be for upgrading minimum PHP to 8.1 🤔

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -13,23 +13,30 @@ on:
 jobs:
     tests:
         runs-on: ubuntu-latest
+        continue-on-error: ${{ matrix.experimental }}
```
Comment (GSadee) [inline .github/workflows/build.yml]: As we would like to add support for SF7 with this PR, I'm not sure if this flag is needed 🤔

File: .github/workflows/build.yml (hunk 2 above)
Comment (loic425) [reply .github/workflows/build.yml:20]: done, I bump php version on Bundle and composer.json

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Loïc! :1st_place_medal:

**Pull request merged**